### PR TITLE
Add perimeter functions

### PR DESCRIPTION
**Pull request by WorldMaker**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -115,19 +115,19 @@ export default class Shape {
...
-  perimiters() {
-    return this.paths.map(path => ClipperLib.Clipper.PerimiterOfPath(path, this.closed))
+  perimeters() {
+    return this.paths.map(path => ClipperLib.Clipper.PerimeterOfPath(path, this.closed))
   }
   
-  totalPerimiter(index) {
-    const perimiter = ClipperLib.Clipper.PerimiterOfPaths(this.paths, this.closed);
-    return perimiter;
+  totalPerimeter(index) {
```
Comment (peteruithoven) [inline src/index.js]: This `index` isn't used? Probably copied from the `perimeter()` function.

File: src/index.js [hunk 2]
```diff
@@ -115,19 +115,19 @@ export default class Shape {
...
-    const perimiter = ClipperLib.Clipper.PerimiterOfPath(path, this.closed);
-    return perimiter;
+    const perimeter = ClipperLib.Clipper.PerimeterOfPath(path, this.closed);
+    return perimeter;
   }
   
-  perimiters() {
-    return this.paths.map(path => ClipperLib.Clipper.PerimiterOfPath(path, this.closed))
+  perimeters() {
+    return this.paths.map(path => ClipperLib.Clipper.PerimeterOfPath(path, this.closed))
```
Comment (peteruithoven) [inline src/index.js]: For consistency, could you add a `;` at the end here? 
We should really add a ESLint profile and probably a pre-commit hook.

**Comment by peteruithoven** [comment]
Thanks for the additions! I added a few notes inline.

**Comment by casperlamboo** [comment]
Looks good! Could you also extend the read me with your additions?

File: src/index.js (hunk 2 above)
Comment (WorldMaker) [inline src/index.js]: Ah, an easy mistake for me to make at this point as my linters all default to [Standard](http://standardjs.com/) these days.

**Comment by WorldMaker** [comment]
Changes have been made based upon the suggestions. :+1:

**Comment by peteruithoven** [comment]
Awesome :tada:

**Pull request merged**